### DISCUSSION TITLE
lesson 3 : identifier not found or not unique

### DISCUSSION THREAD

**Discussion by Magendra-ssn** [smartcontractkit/full-blockchain-solidity-course-py#835] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/85786067/150513218-cb9d7161-075c-493e-a7db-ee1e90cd5c52.png)

// SPDX-Liscense-Identifier: MIT

pragma solidity ^0.6.0;

import "./storage11.sol";

contract storfact
{
    storage11[] public arr;

    function createsf()
    public
    {
        storage11 stor = new storage11();
        arr.push(stor);
    }

}

pls help ! I have also attached a screenshot

**Comment by cromewar** [comment, marked as answer]
Hello @Magendra-ssn 
Why you are using import `"./storage11.sol";`?

as you can see bellow the name is different:
![image](https://user-images.githubusercontent.com/20199180/150534831-f80ae16c-1b14-422a-b4cb-132cbc29611f.png)

1_Storage, so you must use the correct name.

**Reply by Magendra-ssn** [reply]
I was using a .sol outside contracts ( storage11 ). Anyways thanks i got it now, my .sol name and contract name was different that is why i was getting this error. Thanks again.
